lib/srv/term.go

```diff
@@ -560,6 +571,7 @@ func (t *remoteTerminal) PID() int {
 }
 
 func (t *remoteTerminal) Close() error {
+	t.wg.Wait()
```

Comment:
This maybe controversial, but this `t.wg` is not being used anywhere, and from what I see we should wait on it here (the same as `terminal`).